index.html

```diff
@@ -7,139 +7,65 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
     <title>Crossfit Games Bahia</title>
 
+    <link rel="shortcut icon" type="image/png" href="assets/img/favicon.png"/>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&amp;lang=en">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.teal-red.min.css">
-    <link rel="stylesheet" href="styles.css">
-    <style>
-    #view-source {
-      position: fixed;
-      display: block;
-      right: 0;
-      bottom: 0;
-      margin-right: 40px;
-      margin-bottom: 40px;
-      z-index: 900;
-    }
-    </style>
+    <link rel="stylesheet" href="assets/css/styles.css">
+    <link rel="stylesheet" href="assets/css/tablesaw.css">
   </head>
   <body>
     <div class="demo-layout mdl-layout mdl-layout--fixed-header mdl-js-layout mdl-color--grey-100">
       <div class="demo-ribbon"></div>
+
       <main class="demo-main mdl-layout__content">
         <div class="demo-container mdl-grid">
           <div class="center panel demo-content mdl-color--white mdl-shadow--4dp content mdl-color-text--grey-800 mdl-cell mdl-cell--8-col">
-            <h3 class="center title">Leaderboard - Crossfit Games - Bahia</h3>
-
-            <div class="leaderboard-type">
-              <button id="menLeaderboard" class="mdl-button mdl-js-button mdl-button--raised mdl-button--colored">
-                MASCULINO RX
-              </button>
-
-              <button id="womenLeaderboard" class="mdl-button mdl-js-button mdl-button--raised mdl-js-ripple-effect mdl-button--accent">
-                FEMININO RX
-              </button>
-            </div>
 
-            <div class="demo-crumbs mdl-color-text--grey-500 leaderboard-description">
-              Exibindo leaderboard para <strong><span id="type">MASCULINO RX</span></strong>
+            <h3 class="center title">Leaderboard - Crossfit Games 2017 - Bahia</h3>
+
+            <div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect">
+                <div class="mdl-tabs__tab-bar">
+                    <a href="#leaderboard" class="mdl-tabs__tab is-active" onclick="generateLeaderboard('assets/data/women_leaderboard.json')">Feminino RX</a>
```

Comment:
To improve future changes, is better if you define an event listener for the button click instead of use `onclick`